packages/discovery-provider/plugins/pedalboard/apps/relay/package.json

```diff
@@ -6,7 +6,7 @@
     "build": "tsc",
     "build:watch": "tsc --watch",
     "clean": "rm -rf dist",
-    "dev": "nodemon ./dist/index.js",
+    "dev": "concurrently \"npm:build:watch\" \"nodemon ./dist/index.js\"",
```

Comment:
ts wasn't being rebuilt, ts-node-dev also wasn't playing nice with basekit so this gets around that